src/main/java/com/attica/athens/domain/agora/application/AgoraService.java

```diff
@@ -125,6 +129,19 @@ private Category findByCategory(final Long categoryId) {
                 .orElseThrow(() -> new NotFoundCategoryException(categoryId));
     }
 
+    public AgoraTitleResponse getAgoraTitle(final Long agoraId) {
+        Agora agora = agoraRepository.findById(agoraId)
+                .orElseThrow(() -> new NotFoundAgoraException(agoraId));
+
+        return new AgoraTitleResponse(agora.getTitle(), agora.getStatus());
```

Comment:
p2:
- agora 조회하는 부분은 메서드로 분리해도 되겠네요~
- `AgoraTitleResponse`에 `Agora`만 인자로 넘겨도 충분하지 않을까요?

Reply:
분리하겠습니다! 

`agora status`는 dobby님 요구사항으로 추가했습니다 :)

Reply:
@yybmion 두번째에 제안드린 부분은 agora만 dto로 넘겨도 충분하지 않을지에 대한 이야기였어요~
```suggestion
        return new AgoraTitleResponse(agora);
```

Reply:
그렇군요! 근데 제가 AgoraTitleResponse에 String title, AgoraStatus status 이 2가지 인자를 응답하려고 하는데 agora만 넘겨도 되나요??

Reply:
agora를 넘기고 dto안에서 해당 속성을 `getter`로 가져오면 되지 않을까요?

제가 아래 댓글에서는 필요한 데이터만 전달하는 것에 대해 제안을 드려서 조금 헷갈리실 것 같아 정리하겠습니다.
- `service`에서 `repository`로 서비스에서 사용하는 dto를 전달하는 것은 계층간의 결합도가 높아질 수 있습니다.
   필요한 데이터만 전달하면 좋지 않을까요?

- dto를 생성할때에는 객체를 바로 넘기는 것이 개인적으로 더 좋은 것 같아요.
  dto 구조가 변경될 수 있으므로 객체만 전달하면 변경점이 줄어들고, dto는 내부 구조가 노출되어있으므로 결합도를 크게 신경쓰지 않아도 될 것 같아요 ([디미터 법칙](https://dkswnkk.tistory.com/687)도 dto에는 적용되지 않기 때문이에요)
  
  개인적인 의견이라 @yybmion 의 의견도 들어보고 싶네요!

Reply:
민트님이 보내주신 `디미터 법칙` 정말 잘 읽었습니다.

이에 대해 질문이 있어 올립니다.

`디미터법칙` 은 객체가 또 다른 객체에 대해 지나치게 많은 정보를 가지고 있다는 것이 문제였는데 이를 해소시키기 위해 나왔고, 최소한의 .dot을 이용하고 함수를 통해 공개하게 만든다는 것이 핵심이라고 정리할 수 있겠네요. 

앞서 민트님이 말씀하셨다시피 dto는 예외사항으로 내부구조를 외부에 노출하는 것이 당연하여 `디미터법칙`이 적용되지 않는다고 나왔는데, 때문에 그런진 몰라도 지금 해당 문제가 `디미터법칙`과 크게 관련되어 있는게 맞나? 라고 생각이 드네요. 

물론 민트님이 말씀해주신 dto 변경이 일어난다면 만약 service에서 get을 쓰면 변경이 일어나 유지보수가 어려워진다는 단점에는 공감이 갑니다. 따라서 agora 자체를 넘겨주고 dto에서 getter을 통해 관리하면 이후 변경점이 있더라도 dto 부분만 변경하면 된다는 것입니다.

음,,,, 적다 보니 혼란이 와서 이후의 얘기는 좀 더 민트님과 나눠보겠습니다.

결론은 지금 이 문제가 `디미터법칙`이 맞는것인가? 입니다. 

좋은 자료 감사합니다 👍 @Starlight258

Reply:
dto는 디미터 법칙 예외상황인데 제가 괜히 언급해서 헷갈리게 한 것 같네요..! 
오히려 수정하기 전이 디미터 법칙에는 맞는데, dto라 관련이 없는겁니닷

dto에 agora를 넘기면 dto의 내부 구조가 바뀌어도 변경점이 줄어든다.
dto는 결합도가 있을수밖에 없다. 
이 이야기를 하고싶었어요!